Sources/_CJavaScriptKit/_CJavaScriptKit.c

```diff
@@ -35,8 +35,15 @@ void swjs_cleanup_host_function_call(void *argv_buffer) {
 /// Notes: If you change any interface of runtime library, please increment
 /// this and `SwiftRuntime.version` in `./Runtime/src/index.ts`.
 __attribute__((export_name("swjs_library_version")))
-int swjs_library_version() {
+int swjs_library_version(void) {
     return 701;
 }
 
+int _library_features(void);
```

Comment:
Does this addition warrant a version bump?

Reply:
Good catch!